ci/plugins/mzcompose/hooks/command

```diff
@@ -14,7 +14,7 @@ set -euo pipefail
 . misc/shlib/shlib.bash
 
 mzcompose() {
-    bin/ci-builder run stable bin/mzcompose --find "$BUILDKITE_PLUGIN_MZCOMPOSE_COMPOSITION" "$@"
+    bin/ci-builder run stable bin/pyactivate -u -m materialize.cli.mzcompose --find "$BUILDKITE_PLUGIN_MZCOMPOSE_COMPOSITION" "$@"
```

Comment:
It seems funny to do this for mzcompose only. Should we perhaps set `ENV PYTHONUNBUFFERED=1` in the Dockerfile instead?